src/components/project/handlers/apply-approved-changeset-to-project.handler.ts

```diff
@@ -35,17 +44,61 @@ export class ApplyApprovedChangesetToProject
       }
       const { id, createdAt, type, financialReportPeriod, ...actualChanges } =
         changes;
-      await this.projectService.update(
-        {
-          id,
-          ...actualChanges,
-          ...(financialReportPeriod ? { financialReportPeriod } : {}),
-        },
-        event.session,
-        undefined,
-        false
+
+      await Promise.all(
+        keys(actualChanges).map(async (key) => {
+          const query = this.db
+            .query()
+            .match(node('node', 'Project', { id }))
+            // Apply previous label to active prop
+            .apply(addPreviousLabel(key, changesetId))
```

Comment:
Added `previous` label before deactivating active prop because it will be tricky to get deleted prop after applying to deactivate query.